INSTALL_WINDOWS.md

```diff
@@ -1,18 +1,42 @@
 ## Font Bakery Install Instructions for Windows
 
-You'll need Python 3.6 (or newer) to run FontBakery.
+You'll need Python 3.6 (or newer) to run FontBakery, get it from www.python.org.
 
 To install:
 
-    pip install fontbakery;
+    pip install fontbakery
     
 To upgrade:
 
-    pip install --upgrade fontbakery;
+    pip install --upgrade fontbakery
+    
+Should the above commands give you permission errors, try installing FontBakery into your user directory:
+
+    pip install --user fontbakery
+
+You then need to add the directory to your environment path. Follow all ways on https://superuser.com/questions/949560/how-do-i-set-system-environment-variables-in-windows-10 until you get to the dialog where you can set variables either system-wide or for your user account. What you need to do is add `%LOCALAPPDATA%\Python\PythonYOURPYTHONVERSION\Scripts` to your user account's PATH variable, e.g. `%LOCALAPPDATA%\Python\Python36\Scripts` for Python 3.6. 
+
+If this doesn't work for you, wait for the next release of FontBakery after 0.5.1 and use `py -3 -m fontbakery` everywhere you'd use the command `fontbakery`.
```

Comment:
What does this mean? What is different here in the upcoming release?

Reply:
https://github.com/googlefonts/fontbakery/commit/37683adf86a690610448bd0894703130bba7fbf0

This means that FB can be invoked as a module like `python3 -m fontbakery`, circumventing PATH problems because Python will figure them out by itself.

Reply:
oh yeah, now I remember that. Pretty cool :-)

Reply:
The user then either knows what she is doing or is knee-deep in excrement anyway :+1:.